frontend/src/routes/(authenticated)/project/[project_code]/+page.ts

```diff
@@ -106,8 +112,31 @@ export async function load(event: PageLoadEvent) {
   }
 
   event.depends(`project:${projectCode}`);
+  let orgs;
+  if (userIsAdmin) {
+  const orgsPromise = await client.query(graphql(`
```

Comment:
Since you're awaiting the query result, this is not a promise. Let's rename it to `orgsResult`. Same with myOrgsPromise, which should be renamed to `myOrgsResult`.

(Try removing the `await` and hovering over the type of the orgsPromise / orgsResult value. Without the `await`, the type will be `Promise<SomeType>`, where "SomeType" depends on the query. If you put the `await` back, then type becomes just `SomeType`, i.e. it's no longer wrapped in a Promise. It's not always necessary for variable names to reflect the types of what they hold, i.e. if a variable holds a promise you don't **have** to name it fooPromise. But it's a bad idea to name it fooPromise if it does **not** hold a promise.)